### PR TITLE
move to double quotes for all files

### PR DESCRIPTION
**Pull request by peterBrxwn**

### CODE REVIEW

**Comment by peterBrxwn** [comment]
[[RN] Disable typescript linter and enable only esLint linter (10.01)](https://trello.com/c/xbCIyccb/1053-rn-fix-some-linter-issues)

**Review by turtledreams — APPROVED** [review]

File: example/CountlyRNExample/Configuration.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
-import CountlyConfig from 'countly-sdk-react-native-bridge/CountlyConfig';
+import CountlyConfig from "countly-sdk-react-native-bridge/CountlyConfig";
 
-const COUNTLY_SERVER_KEY = 'https://yourdomain.countly';
-const COUNTLY_APP_KEY = 'COUNTLY_APP_KEY';
+const COUNTLY_SERVER_KEY = "https://yourdomain.countly";
+const COUNTLY_APP_KEY = "COUNTLY_APP_KEY";
 
 const countlyConfig = new CountlyConfig(COUNTLY_SERVER_KEY, COUNTLY_APP_KEY).setLoggingEnabled(true); // Enable countly internal debugging logs
 // .setDeviceID(Countly.TemporaryDeviceIDString) // Enable temporary id mode
```
Comment (arifBurakDemiray) [inline example/CountlyRNExample/Configuration.tsx:7]: comment quotes not doubled, should they?

File: Countly.js [hunk 2]
```diff
@@ -1651,26 +1651,26 @@ Countly.presentFeedbackWidgetObject = async function (feedbackWidget, closeButto
...
+        message = "FeedbackWidget type should not be null or empty";
         L.e(`presentFeedbackWidgetObject, ${message}`);
         return message;
     }
-    if (typeof closeButtonText !== 'string') {
-        closeButtonText = '';
-        L.w('presentFeedbackWidgetObject, ' + `unsupported data type of closeButtonText : '${typeof args}'`);
+    if (typeof closeButtonText !== "string") {
+        closeButtonText = "";
+        L.w("presentFeedbackWidgetObject, " + `unsupported data type of closeButtonText : '${typeof args}'`);
```
Comment (arifBurakDemiray) [inline Countly.js:1673]: right part should also be double quoted?

File: Countly.js [hunk 3]
```diff
@@ -1902,22 +1902,22 @@ Countly.setCustomMetrics = async function (customMetric) {
...
     let message = null;
     if (!customMetric) {
-        message = 'customMetric should not be null or undefined';
+        message = "customMetric should not be null or undefined";
         L.e(`setCustomMetrics, ${message}`);
         return message;
     }
-    if (typeof customMetric !== 'object') {
+    if (typeof customMetric !== "object") {
         message = `unsupported data type of customMetric '${typeof customMetric}'`;
```
Comment (arifBurakDemiray) [inline Countly.js:1910]: double quote

File: Countly.js [hunk 4]
```diff
@@ -1902,22 +1902,22 @@ Countly.setCustomMetrics = async function (customMetric) {
...
     }
     const args = [];
     for (const key in customMetric) {
-        if (typeof customMetric[key] === 'string') {
+        if (typeof customMetric[key] === "string") {
             args.push(key.toString());
             args.push(customMetric[key].toString());
         } else {
-            L.w('setCustomMetrics, ' + `skipping value for key '${key.toString()}', due to unsupported data type '${typeof customMetric[key]}'`);
+            L.w("setCustomMetrics, " + `skipping value for key '${key.toString()}', due to unsupported data type '${typeof customMetric[key]}'`);
```
Comment (arifBurakDemiray) [inline Countly.js:1920]: should right part be double quoted?

**Review by arifBurakDemiray — CHANGES_REQUESTED** [review]
in line 640 comments quotes are not doubled, actually most of the comment quotes
`     * @deprecated in 23.02.0 : use 'countlyConfig.enableApm' instead of 'enableApm'.
` like this

https://github.com/Countly/countly-sdk-react-native-bridge/blob/3b8d064e9e75d27d99d555bc380efb791a35e344/Utils.js#L149
This is commented out but not doubled

In Feedback.js,CountlyConfig.js,Countly.js, function docs quotes are also not doubled

**Pull request merged**